plugins/MCWhitelist/__init__.py

```diff
@@ -0,0 +1,192 @@
+        return embed
+
+    @commands.group()
+    async def whitelist(self, ctx: commands.Context):
+        """Manage the MUNCS Craft server whitelist."""
+        if not ctx.invoked_subcommand:
+            whitelisted_account = await self.get_whitelisted_account(ctx.author)
+            if whitelisted_account:
+                embed = await self.account_embed(whitelisted_account)
+                await ctx.send(embed=embed)
+            else:
+                await ctx.send(
+                    "You don't have a Minecraft account associated with yourself yet, run !whitelist add <username>."
+                )
+
+    @whitelist.command(name="add")
+    async def whitelist_add(self, ctx: commands.Context, username: str):
+        """Add users to the MUNCS Craft servers whitelist."""
+        if await self.is_disallowed(ctx.author):
+            await ctx.send(
+                f"You are disallowed from being added to the MUNCS Craft whitelist."
+            )
+            return
+
+        whitelisted_account = await self.get_whitelisted_account(ctx.author)
+        if whitelisted_account:
+            username = whitelisted_account["minecraft_username"]
+            await ctx.send(
+                f"You already have the Minecraft account '{username}' associated with your Discord account, run !whitelist remove first to change/add a different one."
+            )
+            return
+
+        verified_role = self.bot.get_guild(PRIMARY_GUILD).get_role(VERIFIED_ROLE)
+        if verified_role not in ctx.author.roles:
+            await ctx.send(
+                "You must first verify your MUN account using !identity before adding yourself to the whitelist."
+            )
+            return
+
+        mojang_resp = self.mojang_api.info_from_username(username)
+        if not mojang_resp:
+            await ctx.send(
+                f"Error verifying username '{username}' from Mojang, are you sure you typed it correctly?"
+            )
+            return
+
+        if await self.is_minecraft_account_already_associated(username):
+            await ctx.send(
+                f"Username '{username}' is already on the whitelist, reach out to an executive to resolve this issue."
+            )
+            return
+
+        self.whitelist_http_api.add(username)
+
+        new_whitelisted_account = {
+            "discord_id": ctx.author.id,
+            "minecraft_username": username,
+            "minecraft_uuid": mojang_resp["id"],
+        }
+        await self.whitelisted_accounts.insert_one(new_whitelisted_account)
+
+        embed = await self.account_embed(new_whitelisted_account)
+        await ctx.send("Minecraft account whitelisted!", embed=embed)
+
+    @whitelist.command(name="remove")
+    async def whitelist_remove(self, ctx: commands.Context):
+        """Remove users from the MUNCS Craft servers whitelist."""
+        whitelisted_account = await self.get_whitelisted_account(ctx.author)
+        if not whitelisted_account:
```

Comment:
Same here, rather than checking if the value is truthy, you should check if the value is None